### ISSUE TITLE
Clicking "Learn more" without closing the banner

### ISSUE THREAD

**Issue by justanne** [dobarkod/cookie-banner#24]
I was wondering if there is any way to click the **Learn more** link without closing the banner and accepting the cookie..?

**Comment by zytzagoo** [comment]
That's the way it should be by default.

What happens when you download and run the [demo](https://github.com/dobarkod/cookie-banner/blob/master/tests/demo.html)?
Can you provide a link/fiddle/something demonstrating the opposite?

**Comment by justanne** [comment]
It turns out my modification on the src js itself causes it. 

I pulled the latest changes from the repo and customized it again one by one. I actually wanted the **learn more** link to be a popup rather than as href. On the banner I've got 3 buttons (learn more, print, agree) that I enclosed a parent <div> on them and float it on the right side to separate the message from the buttons. This extra div causes the banner to close when clicking one of the buttons.

Currently I was able to implement it without the banner closing, by assigning/using classes on each of the buttons instead. 

Thank you.
